build-process-introduction.md

```diff
@@ -0,0 +1,107 @@
+---
+---
+
+# Build process introduction
+
+## Table of contents
+
+- [Activating Gulp inside the Studio](#activating-gulp-inside-the-studio)
+- [Customize the build process](#customize-the-build-process)
+- [Additional modules](#additional-modules)
+
```

Comment:
@jonathanargentiero for coherence it would be better to use or not `ing` form for all title :wink: `activate` , `customize` or `activating` , `customizing`